src/cli/__tests__/engine.spec.ts

```diff
@@ -147,4 +147,24 @@ describe('Engine', () => {
 
     expect(result).toBeNull();
   });
+
+  it('returns masked string', () => {
+    const engine = new Engine();
+
+    engine.aoo = {
+      tables: {},
+    };
```

Comment:
I think even this should work?

```suggestion
    engine.aoo = {};
```